api/debug.go

```diff
@@ -35,7 +36,7 @@ func (d *DebugAPI) TraceTransaction(
 ) (json.RawMessage, error) {
 	res, err := d.tracer.GetTransaction(hash)
 	if err != nil {
-		return handleError[json.RawMessage](d.logger, err)
+		return handleError[json.RawMessage](d.logger, &metrics.NoopCollector{}, err)
```

Comment:
Should we use `DefaultCollector` in here and below ?

Reply:
yes